### PR TITLE
Add es.danirod.Cartero

### PR DESCRIPTION
**Pull request by danirod**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  Cartero is a graphical HTTP client that can be used as a graphical alternative to curl, and as a development tool to assist during the design, development and test of web APIs.

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. ~If not, I contacted upstream developers about this submission. **Link:**~

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: es.danirod.Cartero.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+id: es.danirod.Cartero
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.rust-stable]
```
Comment (bbhtt) [inline es.danirod.Cartero.yaml:6]:
```suggestion
sdk-extensions:
  - org.freedesktop.Sdk.Extension.rust-stable
```

File: es.danirod.Cartero.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - --socket=fallback-x11
+  - --socket=wayland
+
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+
+modules:
+  - name: blueprint-compiler
+    buildsystem: meson
+    sources:
```
Comment (bbhtt) [inline es.danirod.Cartero.yaml:24]:
```suggestion
    buildsystem: meson
    cleanup:
      - "*"
    sources:
```

**Comment by bbhtt** [comment]
bot, build es.danirod.Cartero

**Comment by danirod** [comment]
I'm installing and testing the build and it looks to work. I can interact with the application, it saves files, it then reads the files back, can double click app data from Files to open them, settings are also kept. Locales are working as well.

**Comment by bbhtt** [comment]
bot, build es.danirod.Cartero

**Pull request closed without merging**